lib/signs/bus.ex

```diff
@@ -135,23 +142,29 @@ defmodule Signs.Bus do
       end
 
     # Update the sign (if appropriate), and record changes in state
-    new_state =
+    state
+    |> then(fn state ->
       if should_update({top, bottom}, current_time, state) do
         MessageQueue.update_sign({pa_ess_loc, text_zone}, top, bottom, 180, :now)
         %{state | current_messages: {top, bottom}, last_update: current_time}
       else
         state
       end
+    end)
+    |> then(fn state ->
+      if should_read(current_time, state) do
+        if audios != [], do: MessageQueue.send_audio({pa_ess_loc, audio_zones}, audios, 5, 60)
```

Comment:
Question: is the value of the timeout taken from TWE? Since I know there is potential for a lot more audio readouts for bus compared to rail, I'm wondering if there's higher chance for audio to timeout. Or do you think each message should be sufficiently below the timeout considering we iterate through the list and process each entry in `audios` separately?

Reply:
Good catch. TWE uses a timeout of 180, so I'll use that instead.